site/src/utils/appearance.ts

```diff
@@ -14,3 +14,18 @@ export const getLogoURL = (): string => {
 		?.getAttribute("content");
 	return c && !c.startsWith("{{ .") ? c : "";
 };
+
+/**
+ * Exposes an easy way to determine if a given URL is for an emoji hosted on
+ * the Coder deployment.
+ *
+ * Helps when you need to style emojis differently (i.e., not adding rounding to
+ * the container so that the emoji doesn't get cut off).
+ */
+export function isEmojiUrl(url: string | undefined): boolean {
+	if (!url) {
+		return false;
+	}
+
+	return url.startsWith("/emojis/");
+}
```

Comment:
hmm. I'm skeptic about adding special handling like this, but I get the motivation.